modules/cloud-run/main.tf

```diff
@@ -120,9 +120,12 @@ resource "google_vpc_access_connector" "connector" {
   max_throughput = var.vpc_connector_create.throughput.max
   min_instances  = var.vpc_connector_create.instances.min
   min_throughput = var.vpc_connector_create.throughput.min
-  subnet {
-    name       = var.vpc_connector_create.subnet.name
-    project_id = var.vpc_connector_create.subnet.project_id
+  dynamic "subnet" {
+    for_each = var.vpc_connector_create == {} ? [] : [""]
+    content {
+      name       = var.vpc_connector_create.subnet.name
+      project_id = var.vpc_connector_create.subnet.project_id
+    }
```

Comment:
without there's a drift is subnet is not specified
```
      + subnet { # forces replacement
          + project_id = (known after apply)
        }
```

Reply:
actually this still doesn't solve the drift 🤔 maybe the issue is in the provider